Libplanet/Store/TrieStateStore.cs

```diff
@@ -90,6 +93,78 @@ public void ForkStates<T>(Guid sourceChainId, Guid destinationChainId, Block<T>
             // Do nothing.
         }
 
+        public void PruneStates(ImmutableHashSet<HashDigest<SHA256>> excludeBlockHashes)
+        {
+            var stopwatch = new Stopwatch();
+            Log.Debug($"Started {nameof(PruneStates)}()");
```

Comment:
I guess it would be better we downgrade its log level to `verbose`

Reply:
Then doesn't it need also new `verbose` message to notice the end? Now it noticed the end like below:

```
Log.Debug(
                "Finished to clean up state hashes. Deleted {DeleteCount}." +
                " elapsed: {ElapsedMilliseconds} ms",
                deleteCount,
                stopwatch.ElapsedMilliseconds);
```

But it seems to need to stay in `debug` level to me. How do you think about this?

Reply:
👌  it looks better